README.md

```diff
@@ -28,6 +28,26 @@ Tracer.initGlobalTracer(LightStep.tracer({
 
 * **Node**: Node versions >= 0.12 are supported.
 
+## LightStep Initialization Options
+
+### Browser-specific
+
+#### `xhr_url_inclusion_patterns RegExp[]`
```

Comment:
I accidentally hit tab-enter while reaching for the backtick key believe it or not! Where was I?

... could get an `operation_name` to substitute for `XMLHttpRequest` in LightStep.